sdk/metricsadvisor/ai-metrics-advisor/test/public/util/recordedClients.ts

```diff
@@ -11,13 +11,6 @@ import {
   MetricsAdvisorClient,
   MetricsAdvisorAdministrationClient
 } from "../../../src";
-import * as dotenv from "dotenv";
-import { isNode } from "@azure/core-http";
-
-if (isNode) {
```

Comment:
we still need this right? maybe have a copy in this package for now.

Reply:
done. can you verify? @jeremymeng